css/custom.css

```diff
@@ -281,12 +281,21 @@ p.card-text {
         width: 100vw;
     }
 
-    .slider_bg_1 {
-        background-image: url("../img/banner/main_banner.jpg") !important;
-        height: 100vh !important;
-        background-position: top !important;
-        /*background-size: 100vh !important;*/
-        background-size: cover !important;
+
+    /*.slider_bg_1 {*/
+    /*    background-image: url("../img/banner/wordless_banner.jpg") !important;*/
+    /*    height: 100vh !important;*/
+    /*    background-position: center !important;*/
+    /*    !*background-size: 100vw !important;*!*/
+    /*}*/
```

Comment:
No point in commenting it out since we can always just get it back using git. Reduces clutter too